dist/origin-web-common.js

```diff
@@ -1425,11 +1425,11 @@ angular.module('openshiftCommonUI')
 
 angular.module('openshiftCommonServices')
   .constant('API_PREFERRED_VERSIONS', {
-      autoscaling:                  {group: 'autoscaling',            resource: 'horizontalpodautoscalers' },
+      horizontalpodautoscalers:     {group: 'autoscaling',            resource: 'horizontalpodautoscalers' },
```

Comment:
nit: alpha

Reply:
lol not `alpha` as in api, `alpha` as in alphabetize.  I will fix :)